### PR TITLE
Merge scopes together when calling the event command

### PR DESCRIPTION
**Pull request by Alex7Li**
This commit updates the event command. The final parameter should include not just those parameters defined in the command, but also those defined by calling the command `measure('set, 'processorName', {paramsTo: 'persist'})`, and those defined in the constructor.

There are some issues with the set command that cause it not to work for now, explained here: https://docs.google.com/document/d/1BRUjcRSOG2cX1okcjbqsXlCM62wozotrvniEYjz736Q/edit#
It should work when data layer helper is updated (https://github.com/google/data-layer-helper/pull/59)

### CODE REVIEW

File: src/config/setup.js [hunk 1]
```diff
@@ -30,10 +30,9 @@ function setupMeasure(dataLayer) {
   const helper = new DataLayerHelper(dataLayer, {'processNow': false});
 
   helper.registerProcessor('config',
-      (eventProcessor, eventOptions, storageProcessor, storageOptions) => {
+      (eventProcessor, eventOptions, storageProcessor, storageOptions) =>
```
Comment (durbanie) [inline src/config/setup.js:33]: Why did you remove the braces?

File: test/unit/config/processEvent_test.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    dataLayer = [];
+    measure = function() {
+      dataLayer.push(arguments);
+    };
+    setupMeasure(dataLayer);
+  });
+
+
+  describe('The call forwarded after a call to measure(`event`, ...)', () => {
+    /* TODO:(wolfblue@): Fix the issue here, it stems from data layer helper
```
Comment (durbanie) [inline test/unit/config/processEvent_test.js:48]: I would strongly prefer that we don't merge commented code, so please uncomment once https://github.com/google/data-layer-helper/pull/59 is merged and DLH is updated.

File: src/config/setup.js (hunk 1 above)
Comment (Alex7Li) [reply src/config/setup.js:33]: There wasn't really any reason, I put them back in so we don't have random changes in the commit history

File: test/unit/config/processEvent_test.js (hunk 2 above)
Comment (Alex7Li) [reply test/unit/config/processEvent_test.js]: It has been uncommented! I will save WIP code locally from now on
